_observability-plugins/data-prepper/get-started.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Get Started
+parent: Data Prepper
+nav_order: 1
+---
+
+# Get started with Data Prepper
+
+Data Prepper is an independent component, not an OpenSearch plugin, that converts data for use with OpenSearch. It's not bundled with the all-in-one OpenSearch installation packages.
+
+## Install Data Prepper
+
+To use the Docker image, pull it like any other image:
+
+```bash
+docker pull opensearchproject/data-prepper:latest
+```
+
+## Define a pipeline
+
+Build a pipeline by creating a pipeline configuration YAML using any of the built-in Data Prepper plugins. 
+
+For more examples and details on pipeline configurations see [Pipelines]({{site.url}}{{site.baseurl}}/observability-plugins/data-prepper/pipelines) guide.
+
+## Start Data Prepper
+
+Run the following command with your pipeline configuration YAML.
+
+```bash
+docker run --name data-prepper -v /full/path/to/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml opensearchproject/opensearch-data-prepper:latest
+```
+
+### Migrating from Logstash
```

Comment:
Minor suggestion, the current document structure could more clearly indicate if you need to define a pipeline and migrate from logstash or if there is a branching path in the steps. When skimming through for the steps I need to follow it reads something like this:

1. install DP
2. define pipeline
3. Run DP
3.1. Migrate from logstash
3.2. Configure Server

Reply:
I agree that it could help but we should keep the documentation consistent with the rest of the website.